src/eko/basis_rotation.py

```diff
@@ -168,3 +171,38 @@ def ad_projectors(nf):
         projs.append(ad_projector(ad, nf))
 
     return np.array(projs)
+
+
+def iuev_labels(nf):
```

Comment:
This name is a bit cryptic: `iuev` is hard to decode, and even if the meaning is clarified in the docstring, it would be better to directly have a telling name.

I would suggest something like:
- `intr_uni_evol_labels`
- `intrinsic_unified_evol_labels`
- `intr_unified_evol_labels`

Choose your favorite combination :)

Reply:
Consider that `intrinsic_evol` is already used in `evoution_operators/flavors.py`, so it would be good also for consistency.